components/button.css

```diff
@@ -8,43 +7,19 @@
     outline: none;
     padding: 0.5rem 1rem;
     cursor: pointer;
-    background-color: var(--basic);
     text-transform: uppercase;
 }
 
-/* ------------------ Contained Buttons ---------------- */
-
-.primary {
-    background-color: var(--primary);
-    color: var(--white);
-}
-
-.secondary {
-    background-color: var(--secondary);
-    color: var(--white);
-}
-
-.success {
-    background-color: var(--success);
-    color: var(--white);
-}
-
-.info {
-    background-color: var(--info);
-    color: var(--white);
-}
-
-.warning {
-    background-color: var(--warning);
-    color: var(--warning-text);
+.button-anchor {
+    color: black;
+    background-color: #efefef;
 }
```

Comment:
you can use CSS variable for color here.

Reply:
Done. Thanks